spec/services/user_signup_spec.rb

```diff
@@ -0,0 +1,138 @@
+require 'spec_helper'
+
+describe UserSignup do
+
+  describe '#sign_up' do
+    let(:stripe_charge) { double(Stripe::Charge) }
+    let(:service) { UserSignup.new(user) }
+
+    before do
+      allow(StripeWrapper::Charge).to receive(:create).and_return stripe_charge
+      allow(stripe_charge).to receive_messages(succesful?: true, customer_id: "ABC123")
+    end
+
+    context 'with valid input' do
+      let(:user) { Fabricate.build(:user) }
+
+      it 'sets the status to success' do
+        service.sign_up
```

Comment:
ah because the stubs don't need the stripe token...

it seems redundant to me to pass in a stripe_token when it's really not needed. Another programmer might even think that this stripe_token may influence the test outcome. 

I've added passing the stripe_token to the UserSignupSpec but I think this may cause confusion. What do you think?